lib/mongoid/rspec.rb

```diff
@@ -29,10 +29,11 @@
 require 'matchers/be_dynamic_document'
 require 'matchers/be_stored_in'
 require 'matchers/have_field'
+require 'matchers/indexes/have_index_for'
 if Mongoid::Compatibility::Version.mongoid4_or_newer?
```

Comment:
Nitpicking, but it's probably better to check whether it's `mongoid3?` and switch the `if` around. It's a more exact `if`.